### PR TITLE
fix debugger property view

### PR DESCRIPTION
**Pull request by epascal**
on ios13 the format of debugger property inspector has changed this fixes the problem.

### CODE REVIEW

**Comment by epascal** [comment]
Two quite dirty fixes that helps a lot to debug, console.log is not showing not enough knowledge to understand why then I change them to info and it works
The debugger inspector message have changed of format, this fix will allow to display variables in the debugger.

**Comment by auchenberg** [comment]
Thanks for helping with this @epascal! I think these changes belongs to a ISO13 specific adaptor, and we have a pattern for this in previous versions of the protocol, but not for iOS13.

If will require you moving this logic to a new file `ios13.js` and follow this pattern https://github.com/RemoteDebug/remotedebug-ios-webkit-adapter/blob/aa5c6907f255bb222e35fa333227830bc9612d74/src/adapters/iosAdapter.ts#L157

File: src/protocols/ios/ios.ts [hunk 1]
```diff
@@ -592,10 +592,11 @@ export abstract class IOSProtocol extends ProtocolAdapter {
         let type;
         if (message.type === 'log') {
             switch (message.level) {
-                case 'log': type = 'log'; break;
+                case 'log': type = 'info'; break;
```
Comment (auchenberg) [inline src/protocols/ios/ios.ts:595]: Is the intention to "mute" the console.log messages or what's the intent of mapping log to info?

Comment (epascal) [reply src/protocols/ios/ios.ts:595]: Apparently the devtools won't show log type messages only info, warning and error.
I debugged devtools to understand, the messages are received, stored in the console array but not displayed, when they are transformed into info they are displayed. 
I didn't find the reason why log type message are filtered, maybe the label is wrong.
I know it's a hack, but very useful.

**Comment by auchenberg** [comment]
The latest commit seems to add all of the build output?

**Comment by epascal** [comment]
> 
> 
> The latest commit seems to add all of the build output?

Sorry wasn't intended to be included in the pull request

**Comment by niksour** [comment]
Hi

**Comment by auchenberg** [comment]
This project is now super-seeded by https://inspect.dev/ – a new developer tool for macOS and Windows to inspect and debug your web apps and websites in Safari and WebViews on iOS devices.

RemoteDebug iOS WebKit Adapter is not proactively maintained or extended.

**Pull request closed without merging**